webapp/src/routes/Home/ShowOffers.js

```diff
@@ -0,0 +1,345 @@
+  },
+  list: {
+    width: "100vw",
+  },
+  listItem: {
+    width: "100%",
+    backgroundColor: "white"
+  }
+  , secondaryIconList: {
+    color: "rgba(0, 0, 0, 0.6)",
+    width: 20,
+    height: 20
+  },
+  listItemPrimaryText: {
+    color: "rgba(0, 0, 0, 0.87)",
+    fontFamily: "Roboto",
+    fontSize: "16px",
+    fontWeight: "normal",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: 1.5,
+    letterSpacing: "0.15px",
+  },
+  listItemSecondaryText: {
+    color: "color: rgba(0, 0, 0, 0.6)",
+    fontFamily: "Roboto",
+    fontSize: "14px",
+    fontWeight: "normal",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: 1.43,
+    letterSpacing: "0.25px",
+  },
+  modal: {
+    position: 'absolute',
+    width: 400,
+    border: '0px',
+    backgroundColor: '#FFF',
+    boxShadow: theme.shadows[5],
+    padding: theme.spacing(2, 4, 3)
+  },
+  modalTitle: {
+    fontSize: 30,
+    marginBottom: theme.spacing(4)
+  },
+  inputStyle: {
+    width: '100%',
+    marginBottom: 15
+  },
+  infoText: {
+    fontSize: 30,
+    marginBottom: theme.spacing(4)
+  },
+  offersGridContainer: {
+    width: "100%",
+    marginTop: 15,
+    marginBottom: 15
+  },
+  cardRoot: {
+    position: "relative",
+    width: "100%",
+    height: "145px",
+    padding: 10
+  },
+  cardHeader: {
+    padding: 0,
+    margin: 0,
+    width: "100%",
+    display: "flex",
+    flexDirection: 'row',
+    alignItems: 'flex-start',
+  },
+  cardAvatar: {
+    height: "40px",
+    width: "40px",
+  },
+  cardTitleContainer: {
+    width: "60%"
+  },
+  cardTitle: {
+    marginLeft: 7,
+    marginTop: 10,
+    width: "100%",
+    fontFamily: "Roboto",
+    fontsize: "16px",
+    fontweight: "normal",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: "1.5",
+    letterSpacing: "0.15px",
+    textAlign: "left",
+    color: "rgba(0, 0, 0, 0.87)",
+  },
+  cardIconOffer: {
+    position: "absolute",
+    color: "rgba(0, 0, 0, 0.6)",
+    width: 20,
+    height: 20,
+    top: 20,
+    right: 15
+  },
+  cardContent: {
+    marginTop: 10,
+    padding: 0,
+  },
+  cardContentText: {
+    fontFamily: "Roboto",
+    fontsize: "14px",
+    fontweight: "normal",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: "1.43",
+    letterSpacing: "0.25px",
+    textAlign: "left",
+    color: "rgba(0, 0, 0, 0.6)",
+  },
+  cardActions: {
+    padding: 0,
+    width: "100%",
+    display: "flex",
+    flexDirection: 'column',
+    alignItems: 'flex-end',
+  },
+  cardActionButton: {
+    position: "absolute",
+    bottom: 10,
+    right: 15,
+    fontFamily: "Roboto",
+    fontsize: "14px",
+    fontweight: "normal",
+    fontStretch: "normal",
+    fontStyle: "normal",
+    lineHeight: "1.14",
+    letterSpacing: "1",
+    textAlign: "center",
+    color: "#121212",
+  }
+}))
+
+
+const ShowOffers = ({ offers, loading, isDesktop }) => {
+  const classes = useStyles()
+
+  const LoadOffers = () => {
```

Comment:
When the components only has a return in its JSX content, you can do something like

```
const OfferItem = (props) => (
    JSX HERE
)
```